### PR TITLE
Create IndustryStandards.md

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Replaces #595

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
> I don't understand why you mentioned BIPs in this document. Don't we have a document for that already?

Sorry, I took the template from BIPs.md

File: docs/using-wasabi/IndustryStandards.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Supported Standards
+
+- [What is supported today](IndustryStandards.md#what-is-supported-today)
+- [What will be supported in #twoweeks](IndustryStandards.md#what-will-be-supported-in-twoweeks)
+- [What is not supported](IndustryStandards.md#what-is-not-supported)
+
+---
+
+Wasabi Wallet strives toward establishing solid industry best practices and standards.
+Here is a list of all the supported and integrated Standards:
```
Comment (yahiheb) [inline docs/using-wasabi/IndustryStandards.md]:
```suggestion
Here is a list of all the supported and integrated standards:
```

**Comment by yahiheb** [comment]
All of this could have been done in one commit.
We should start squashing commits, to not manipulate the docs bounty. (which wrongly uses the number of commits as the reference for the job done).

File: docs/using-wasabi/IndustryStandards.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+:::details
+### Hardware Wallet Interface
+
+[Hardware Wallet Interface](https://github.com/bitcoin-core/HWI)
+:::
+
+## What will be supported in #twoweeks
+
+## What is not supported
```
Comment (MaxHillebrand) [inline docs/using-wasabi/IndustryStandards.md]:
```suggestion
```

File: docs/using-wasabi/IndustryStandards.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "A collection and explanation of all the Industry Standards that are supported by Wasabi. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
+}
+---
+
+# Supported Standards
+
+- [What is supported today](IndustryStandards.md#what-is-supported-today)
+- [What will be supported in #twoweeks](IndustryStandards.md#what-will-be-supported-in-twoweeks)
+- [What is not supported](IndustryStandards.md#what-is-not-supported)
+
```
Comment (MaxHillebrand) [inline docs/using-wasabi/IndustryStandards.md]:
```suggestion
[[toc]]

```

File: docs/using-wasabi/IndustryStandards.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+# Supported Standards
+
+[[toc]]
+
+---
+
+Wasabi Wallet strives toward establishing solid industry best practices and standards.
+Here is a list of all the supported and integrated standards:
+
+## What is supported today
```
Comment (yahiheb) [inline docs/using-wasabi/IndustryStandards.md]:
```suggestion
```

File: docs/using-wasabi/IndustryStandards.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+:::details
+### Hardware Wallet Interface
```
Comment (yahiheb) [inline docs/using-wasabi/IndustryStandards.md]:
```suggestion
## Hardware Wallet Interface
```

**Pull request merged**